src/main/data/Sort.ts

```diff
@@ -0,0 +1,5 @@
+export enum Sort {
+  DISTANCE = 'Distance',
+  POPULARITY = 'Popularity',
+  RECENCY = 'Recency',
+}
```

Comment:
Maybe `SortOptions` and some documentation for what this means?

Reply:
Oooh yes I can add this!